concourse/pipelines/guest-package-build.yaml

```diff
@@ -1596,6 +1596,398 @@ jobs:
+      git_ref: ((.:commit-sha))
+  - put: compute-image-windows-tag
+    params:
+      name: package-version/version
+      tag: package-version/version
+      commitish: compute-image-windows/.git/ref
+  - task: compute-image-windows-publish-coverage
+    file: guest-test-infra/concourse/tasks/publish-coverage.yaml
+    vars:
+      package_name: "google-compute-image-windows"
+      coverage_percent: ((.:coverage-percent))
+  on_success:
+    task: success
+    file: guest-test-infra/concourse/tasks/publish-job-result.yaml
+    vars:
+      pipeline: "guest-package-build"
+      job: "build-compute-image-windows"
+      result_state: "success"
+      start_timestamp: ((.:start-timestamp-ms))
+  on_failure:
+    task: failure
+    file: guest-test-infra/concourse/tasks/publish-job-result.yaml
+    vars:
+      pipeline: "guest-package-build"
+      job: "build-compute-image-windows"
+      result_state: "failure"
+      start_timestamp: ((.:start-timestamp-ms))
+
+- name: upload-windows-certgen-staging
+  plan:
+  - in_parallel:
+      steps:
+      - get: compute-image-windows-tag
+        passed: [build-compute-image-windows]
+      - get: guest-test-infra
+      - get: compute-image-windows
+  - task: generate-timestamp
+    file: guest-test-infra/concourse/tasks/generate-timestamp.yaml
+  - load_var: start-timestamp-ms
+    file: timestamp/timestamp-ms
+  - load_var: package-version
+    file: compute-image-windows-tag/version
+  - task: generate-build-id
+    file: guest-test-infra/concourse/tasks/generate-build-id.yaml
+    vars:
+      prefix: ""
+  - load_var: build-id
+    file: build-id-dir/build-id
+  - task: upload-certgen-windows-to-staging
+    file: guest-test-infra/concourse/tasks/gcloud-package-operation.yaml
+    vars:
+      package_path: compute-image-windows/certgen.x86_64.((.:package-version)).0@1.goo
+      universe: cloud-yuck
+      repo: certgen
+    params:
+      TYPE: uploadToStaging
+  on_success:
+    task: success
+    file: guest-test-infra/concourse/tasks/publish-job-result.yaml
+    vars:
+      pipeline: "guest-package-build"
+      job: "upload-windows-certgen-staging"
+      result_state: "success"
+      start_timestamp: ((.:start-timestamp-ms))
+  on_failure:
+    task: failure
+    file: guest-test-infra/concourse/tasks/publish-job-result.yaml
+    vars:
+      pipeline: "guest-package-build"
+      job: "upload-windows-certgen-staging"
+      result_state: "failure"
+      start_timestamp: ((.:start-timestamp-ms))
+
+- name: promote-windows-certgen-stable
+  plan:
+  - get: compute-image-windows-tag
+    passed: [upload-windows-certgen-staging]
+  - get: compute-image-windows
+    params: {fetch_tags: true}
+  - get: guest-test-infra
+  - task: generate-timestamp
+    file: guest-test-infra/concourse/tasks/generate-timestamp.yaml
+  - load_var: start-timestamp-ms
+    file: timestamp/timestamp-ms
+  - task: promote-certgen-windows-x64-stable
```

Comment:
nit, this is the only place you included '-x64'

Reply:
Removed